package.json

```diff
@@ -72,6 +73,7 @@
   "devDependencies": {
     "cypress": "^5.1.0",
     "eslint-plugin-jsx-a11y": "^6.3.1",
+    "node-sass": "^4.14.1",
```

Comment:
Tulisko olla dependencien puolella?

Reply:
I think node-sass is not included in the bundle like React or HDS. That is why I put it into devDependencies.